webhook/resources/backingimage/mutator.go

```diff
@@ -102,12 +103,28 @@ func (b *backingImageMutator) Create(request *admission.Request, newObj runtime.
 		patchOps = append(patchOps, `{"op": "replace", "path": "/spec/disks", "value": {}}`)
 	}
 
+	// Handle Spec.DiskFileSpecMap
+	if backingImage.Spec.DiskFileSpecMap == nil {
+		patchOps = append(patchOps, `{"op": "replace", "path": "/spec/disks", "value": {}}`)
+	}
+
 	longhornLabels := types.GetBackingImageLabels()
 	patchOp, err := common.GetLonghornLabelsPatchOp(backingImage, longhornLabels, nil)
 	if err != nil {
 		err := errors.Wrapf(err, "failed to get label patch for backingImage %v", backingImage.Name)
 		return nil, werror.NewInvalidError(err.Error(), "")
 	}
+
+	if backingImage.Spec.MinNumberOfCopies == 0 {
+		minNumberOfCopies, err := b.getDefaultMinNumberOfBackingImageCopies()
+		if err != nil {
+			err = errors.Wrap(err, "BUG: cannot get valid number for setting default min number of backing image copies")
```

Comment:
Should we need "BUG"?

```suggestion
			err = errors.Wrap(err, "failed to get valid number for setting default min number of backing image copies")
```

Reply:
fixed.